src/main/java/net/sf/jabref/logic/exporter/ModsExportFormat.java

```diff
@@ -1,58 +1,474 @@
-                trans.setOutputProperty(OutputKeys.INDENT, "yes");
-                trans.transform(source, result);
-            } catch (TransformerException | IllegalArgumentException | TransformerFactoryConfigurationError e) {
-                throw new Error(e);
+                for (Map.Entry<String, String> entry : fieldMap.entrySet()) {
+                    String key = entry.getKey();
+                    String value = entry.getValue();
+
+                    switch (key) {
+
+                    case FieldName.AUTHOR:
+                        handleAuthors(mods, value);
+                        break;
+                    case "affiliation":
+                        addAffiliation(mods, value);
+                        break;
+                    case FieldName.ABSTRACT:
+                        addAbstract(mods, value);
+                        break;
+                    case FieldName.TITLE:
+                        addTitle(mods, value);
+                        break;
+                    case FieldName.LANGUAGE:
+                        addLanguage(mods, value);
+                        break;
+                    case FieldName.LOCATION:
+                        addLocation(mods, value);
+                        break;
+                    case FieldName.URL:
+                        addUrl(mods, value);
+                        break;
+                    case FieldName.NOTE:
+                        addNote(mods, value);
+                        break;
+                    case FieldName.KEYWORDS:
+                        addKeyWords(mods, value);
+                        break;
+                    case FieldName.VOLUME:
+                        addDetail(FieldName.VOLUME, value, partDefinition);
+                        break;
+                    case FieldName.ISSUE:
+                        addDetail(FieldName.ISSUE, value, partDefinition);
+                        break;
+                    case FieldName.PAGES:
+                        addPages(partDefinition, value);
+                        break;
+                    case FieldName.URI:
+                        addIdentifier(FieldName.URI, value, mods);
+                        break;
+                    case FieldName.ISBN:
+                        addIdentifier(FieldName.ISBN, value, mods);
+                        break;
+                    case FieldName.ISSN:
+                        addIdentifier(FieldName.ISSN, value, mods);
+                        break;
+                    case FieldName.DOI:
+                        addIdentifier(FieldName.DOI, value, mods);
+                        break;
+                    case FieldName.PMID:
+                        addIdentifier(FieldName.PMID, value, mods);
+                        break;
+                    case FieldName.JOURNAL:
+                        addJournal(value, relatedItem);
+                        break;
+                    default:
```

Comment:
No! Codecov will complain about a case with no default. There always shoold be a default in a switch case